app/Import/ImportExceptionData.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace App\Import;
```

Comment:
I'd move this exception to the exception namespace. We can keep the "import" subfolder, so we don't have one bloated folder.

Maybe it also makes sense to move this into the `ImportException.php` file?